include/deal.II/hp/fe_collection.h

```diff
@@ -274,6 +274,45 @@ namespace hp
     unsigned int
     find_least_face_dominating_fe(const std::set<unsigned int> &fes) const;
 
+    /**
+     * Try to find a most face dominating finite element inside the subset of
+     * fe_indices @p fes as part of this FECollection. For example, if an
+     * FECollection consists of `{FE_Q(1),FE_Q(2),FE_Q(3),FE_Q(4)}` elements
+     * and we are looking for the most face dominating finite element among the
+     * last two elements of this collection (i.e., @p fes is `{2,3}`), then the
+     * answer is FE_Q(4) and therefore this function will return its index in
+     * the FECollection, namely `3`.
+     *
+     * This function differs from find_least_face_dominating_fe() in such a way
+     * that it only considers the subset @p fes, instead of finding a dominating
```

Comment:
maybe elaborate a bit (reword to your liking):

... it looks for the most face dominating FE within the given subset @p fes, instead of finding the least dominating FE within the whole collection of a given subset @p fes.